libc/CMakeLists.txt

```diff
@@ -844,6 +844,13 @@ elseif (CMAKE_C_COMPILER_ID MATCHES Clang OR USE_CLANGW)
     -Wno-unneeded-internal-declaration)
 endif ()
 
+# Disable optimizations for twalk.c to avoid Clang compiler bug
```

Comment:
Consider add more context to the bug (e.g., the clang version, a link to the bug if any).
If it's version-specific, please use `CMAKE_CXX_COMPILER_VERSION` for the check. See the following example.
https://github.com/openenclave/openenclave/blob/3cf566a8d7043855c9d298d25278f5a14ac015d7/tests/libcxx/enc/CMakeLists.txt#L150